test/StringUtility.test.cpp

```diff
@@ -4,6 +4,24 @@
 
 using namespace StringUtility;
 
+TEST(StringUtility, ConvertToUpperInPlace)
+{
+	std::string str("abc");
+	ConvertToUpperInPlace(str);
+
+	EXPECT_EQ("ABC", str);
+}
+
+TEST(StringUtility, ConvertToUpper)
+{
+	std::string str("abc");
+	std::string newStr = ConvertToUpper("abc");
+
+	EXPECT_EQ("ABC", newStr);
+	EXPECT_NE(str, newStr);
+}
```

Comment:
Was curious if you'd meant to write `ConvertToUpper(str)`. That would make the final expectation make a bit more sense.

I really like the structure of the new unit tests.

Reply:
Oh, yeah good catch. I'll fix shortly.